code/modules/mob/death.dm

```diff
@@ -80,6 +80,12 @@
 		healths.icon_state = "health6"
 
 	timeofdeath = world.time
+	if (isanimal(src))
+		set_death_time(ANIMAL, world.time)
+	else if (ispAI(src) || isdrone(src))
+		set_death_time(MINISYNTH, world.time)
+	else if (isliving(src))
```

Comment:
If crew is fallback, change this to `else` as it is in the other file as well. That way, _at least_ this one will be set, if none other is.